crates/lib/tests/prng_test_cases.json

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
These are values that have been validated as final memory read off Rigetti QPUs.